dockerfile_parse/parser.py

```diff
@@ -186,6 +193,18 @@ def _rstrip_backslash(l):
             return l
 
         instructions = []
+        # Handle inherited parent envs
+        for key, val in self.parent_env.items():
+            instructions.append(
```

Comment:
Right, but how else can we possibly parse the inherited ENVs if they are not present?